dev-lang/ghc/ghc-9.6.4.ebuild

```diff
@@ -523,35 +422,6 @@ src_prepare() {
 
 	ghc_setup_cflags
 
-	if ! use ghcbootstrap; then
-		# The switch to gpkg binaries means that they are unpacked in the wrong
-		# location. They are now unnpacked in ${PF}/image and need to be
-		# copied so that usr/ is in $WORKDIR.
-		mv -v "${WORKDIR}/${PN}-${BIN_PVR}/image/usr" "${WORKDIR}" || die
-
-		if [[ ${CHOST} != *-darwin* && ${CHOST} != *-solaris* ]]; then
-			# Modify the wrapper script from the binary tarball to use GHC_PERSISTENT_FLAGS.
-			# See bug #313635.
-			sed -i -e "s|\"\$topdir\"|\"\$topdir\" ${GHC_PERSISTENT_FLAGS}|" \
-				"${WORKDIR}/usr/bin/ghc-${BIN_PV}"
-
-			# allow hardened users use vanilla binary to bootstrap ghc
-			# ghci uses mmap with rwx protection at it implements dynamic
-			# linking on it's own (bug #299709)
-			pax-mark -m "${WORKDIR}/usr/$(get_libdir)/${PN}-${BIN_PV}/bin/ghc"
-		fi
-
-		# Make GHC's settings file comply with user's settings
-        GHC_SETTINGS="${WORKDIR}/usr/$(get_libdir)/${PN}-${BIN_PV}/lib/settings"
-        sed -i "s/,(\"C compiler command\", \".*\")/,(\"C compiler command\", \"$(tc-getCC)\")/" "${GHC_SETTINGS}" || die
-        sed -i "s/,(\"C++ compiler command\", \".*\")/,(\"C++ compiler command\", \"$(tc-getCXX)\")/" "${GHC_SETTINGS}" || die
-        sed -i "s/,(\"Haskell CPP command\", \".*\")/,(\"Haskell CPP command\", \"$(tc-getCC)\")/" "${GHC_SETTINGS}" || die
-        sed -i "s/,(\"ld command\", \".*\")/,(\"ld command\", \"$(tc-getLD)\")/" "${GHC_SETTINGS}" || die
-        sed -i "s/,(\"Merge objects command\", \".*\")/,(\"Merge objects command\", \"$(tc-getLD)\")/" "${GHC_SETTINGS}" || die
-        sed -i "s/,(\"ar command\", \".*\")/,(\"ar command\", \"$(tc-getAR)\")/" "${GHC_SETTINGS}" || die
-        sed -i "s/,(\"ranlib command\", \".*\")/,(\"ranlib command\", \"$(tc-getRANLIB)\")/" "${GHC_SETTINGS}" || die
-	fi
```

Comment:
@vimproved 

Did you remove this by mistake?

Reply:
The configure script for the binary GHC does this already.